examples/boost/CMakeLists.txt

```diff
@@ -4,19 +4,34 @@ project(CPMExampleBoost)
 
 # ---- Create binary ----
 
-add_executable(CPMExampleBoost main.cpp)
+add_executable(${PROJECT_NAME} main.cpp)
 target_compile_features(CPMExampleBoost PRIVATE cxx_std_17)
 
 # ---- Dependencies ----
 
 include(../../cmake/CPM.cmake)
 
 CPMAddPackage(
-  NAME Boost
-  VERSION 1.84.0
-  URL https://github.com/boostorg/boost/releases/download/boost-1.84.0/boost-1.84.0.tar.xz
-  URL_HASH SHA256=2e64e5d79a738d0fa6fb546c6e5c2bd28f88d268a2a080546f74e5ff98f29d0e
-  OPTIONS "BOOST_ENABLE_CMAKE ON" "BOOST_INCLUDE_LIBRARIES container\\\;asio" # Note the escapes!
```

Comment:
This is a simple, _secure_, working example that demonstrates including `container` and `asio`. For me security and simplicity are important.

Reply:
I agree that this is important.